narwhals/_arrow/expr.py

```diff
@@ -171,6 +171,9 @@ def all(self) -> Self:
     def sum(self) -> Self:
         return reuse_series_implementation(self, "sum", returns_scalar=True)
 
+    def drop_nulls(self) -> Self:
+        return reuse_series_implementation(self, "drop_nulls", returns_scalar=True)
```

Comment:
`returns_scalar` is for when it returns a single value - here, it may return many. so, you should remove it